### DISCUSSION TITLE
Tervetuloa Koha-translations-keskusteluun

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha-translations#16] [Tiedotteet]
Täällä voimme keskustella Kohan suomennoksissa käytettävistä termeistä. Listaus termeistä löytyy repositorion wikistä.

**Comment by paivito** [comment]
Hei, tällainen yleinen käännöksiin liittyvä huomio. Tuolla Termistö-listassa "material type" on käännetty "aineistotyypiksi". Olen kuitenkin törmännyt Kohassa toiseen käännökseen, jolle näytetään annetun sama merkitys, eli "record-level itemtype" on käännetty niinikään aineistotyypiksi. Tuo recor-level itemtype tulee esille, kun esim. palauttaa kirjan virkailijaliittymässä, ks. liite:
[esim.pdf](https://github.com/KohaSuomi/Koha-translations/files/12416383/esim.pdf)

**Reply by AnneliO** [reply]
Juu, ne on käytännössä sama asia. Tuo material type taitaa olla Koha-Suomen omaa perua, kun otimme käyttöön tietueisiin 942c-kenttään auktorisoidut arvot (MTYPE), jotka vastaavat Finnan aineistotyyppejä.
